blueprints/ember-cli-typescript/files/app/config/environment.d.ts

```diff
@@ -9,4 +9,7 @@ export default config;
  */
 declare namespace config {
   export var environment: any;
+  export var modulePrefix: String;
```

Comment:
`String` should be `string`  -- i'm new to typescript and made this mistake :)

https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html